src/main/java/net/imglib2/type/logic/BitType.java

```diff
@@ -326,7 +326,7 @@ public String toString()
 	@Override
 	public Fraction getEntitiesPerPixel()
 	{
-		return new Fraction();
+		return new Fraction(1, 64);
```

Comment:
Awesome catch!

Could you please add a regression test that calls `ArrayImgs.bits` and then rips out the `long[]` (via `getCurrentStorageArray()`) and asserts that it is the right length?

Reply:
https://github.com/gab1one/imglib2-bittype/blob/master/src/test/java/org/knime/knip/ArrayImgOversizeTest.java Like this one?

Reply:
Something like that, yeah! Makes sense to add it here, right?